You are driving from Dhaka city to Jessore city and then going to Khulna.
You want to know how much total distance you will be travelling.

In [6]:
dha_jes=245 #km
jes_khl=70 #km
total_distance=dha_jes+jes_khl
total_distance

315

BUS speed 65 km/h and it does not take any break. How much time do we need from dhaka to khulna by bus?

Formulla: time=total distance/speed

In [12]:
speed=65 #km/h
TotalTime=total_distance/speed
print(round(TotalTime,2))

4.85


##### it takes around 4.85 hrs to reach the khulna by bus.